core/base/src/utils/encoding.ts

```diff
@@ -1,5 +1,7 @@
 import { base16, base64, base58 } from '@scure/base';
 
+export { bech32 } from '@scure/base';
```

Comment:
I expect this will work fine but can we just export it the same way as others?

Reply:
The issue is that bech32 is rather different to the rest because it's not a simple one to one conversion. It takes the human readable descriptor, the actual data, and then it computes a payload (and vice versa).

I'm already using the slightly awkward `decodeWithBytes` method (or whatever its exact name was).

I'd have made it uniform if I had seen a straigh-forward way how.